byron/ledger/executable-spec/src/Cardano/Ledger/Spec/STS/UTXO.hs

```diff
@@ -51,6 +51,10 @@ instance STS UTXO where
   type Environment UTXO = UTxOEnv
   type State UTXO = UTxOState
   type Signal UTXO = Tx
+
+  -- | These `PredicateFailure`s are all "throwable". The disjunction of the
+  --   rules' preconditions is not `True` - the `PredicateFailure`s represent
```

Comment:
Maybe we'd want to put this comment in one place, like in the class definition.

Reply:
Although I see now that we cannot state a general principle or rule in the class definition, so maybe it's ok to have them here.

Reply:
Is this is somewhat good statement of the general principle? https://github.com/input-output-hk/cardano-ledger-specs/pull/772/commits/fac7363f75d7215410bb27c7a1708e6c0392a88d#diff-e5b2ef76467b113fba0e7e14fa3216ecR89

Reply:
It looks quite clear to me :) Thanks.